website/content/docs/configuration/tls.mdx

```diff
@@ -23,6 +23,8 @@ tls {
 ~> Incorrect configuration of the TLS configuration can result in failure to
 start the Nomad agent.
 
+~> When enabling `rpc=true`, all RPC endpoints will use mutual authentication (mTLS)
+
```

Comment:
We can probably just put this note inside the text of the `rpc` field below. No need for it to be a warn box.